templates/cluster-template-azure.yaml

```diff
@@ -0,0 +1,143 @@
+# Based on: https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/download/v1.5.3/cluster-template.yaml
+
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: AzureCluster
+    name: ${CLUSTER_NAME}
+  controlPlaneRef:
+    kind: MicroK8sControlPlane
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    name: ${CLUSTER_NAME}-control-plane
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureCluster
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: default
```

Comment:
remove the namespace, this is decided during `clusterctl generate`. Also in other places (e.g. the AzureMachineTemplates)

Reply:
For now, it seems to work. I'll remove the hardcoded namespaces in a separate PR.

Reply:
does it work if you deploy a cluster in a different namespace? e.g. `clusterctl generate -n my-namespace ...`?

Reply:
Fixed.